scalafmt-core/shared/src/main/scala/org/scalafmt/config/ScalafmtConfig.scala

```diff
@@ -327,13 +328,14 @@ object ScalafmtConfig {
     )
   }.map { case (k, v) => k.toLowerCase -> v }
 
-  def conservativeRunner: ScalafmtRunner = default.runner.copy(
-    optimizer = default.runner.optimizer.copy(
-      // The tests were not written in this style
-      forceConfigStyleOnOffset = 500,
-      forceConfigStyleMinArgCount = 5
+  def conservativeRunner: ScalafmtRunner =
```

Comment:
while i am ok with this, it's the result of the fix to `alwaysBeforeMultilineDef` (which wasn't working properly before). if you wish, we can disable the flag, see if that reduces diff.

Reply:
No, I am ok with it